webapp/locales/de.json

```diff
@@ -279,8 +279,9 @@
         },
         "comment": {
             "submit": "Kommentiere",
-            "submitted": "Kommentar gesendet",
-            "updated": "Änderungen gespeichert"
+            "submitted": "Kommentar gesendet!",
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-01-09T08:08:10Z" title="Thursday, January 9th 2020, 9:08:10 am +01:00">Jan 9, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -269,7 +269,8 @@
         "comment": {
             "submit": "Kommentiere",
             "submitted": "Kommentar Gesendet",
-            "updated": "Änderungen gespeichert"
+            "updated": "Änderungen gespeichert",
+            "answered": "answered"
```


Please translate.
I would name this `"answer": „Antworten“`.

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2020-01-10T07:19:44Z" title="Friday, January 10th 2020, 8:19:44 am +01:00">Jan 10, 2020</time>_

---


ok